index.html

```diff
@@ -9,44 +9,272 @@
+      <h1 class="about-title">About me</h1>
+      <p class="about-par"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and<br> have a project you need coded, 
+        don’t hesitate to contact me.</p>
+      <button class="btn1">Get my resume</button>
+    </div>
+
+    <div class="all-about">
+<!-- card -->
+<div class="card about-card">
+  <img class="about-img" src="img3.png" alt="project">
+  <div class="card-content about-content">
+    <h2 class="card-title">languages</h2>
+    <ul class="skills">
+      <li class="skill skill-about">JavaScript</li>
+      <li class="skill skill-about">Ruby</li>
+      <li class="skill skill-about">HTML</li>
+      <li class="skill skill-about">CSS</li>
+    </ul>
+    
+  </div>
+</div>
+ <!-- card -->
+ <div class="card about-card">
+  <img class="about-img" src="img1.png" alt="project">
+  <div class="card-content about-content">
+    <h2 class="card-title">languages</h2>
+    <ul class="skills">
+      <li class="skill skill-about">JavaScript</li>
+      <li class="skill skill-about">Ruby</li>
+      <li class="skill skill-about">HTML</li>
+      <li class="skill skill-about">CSS</li>
+    </ul>
+    
+  </div>
+</div>
+ <!-- card -->
+ <div class="card about-card">
+  <img class="about-img" src="img2.png" alt="project">
+  <div class="card-content about-content">
+    <h2 class="card-title">languages</h2>
+    <ul class="skills">
+      <li class="skill skill-about">JavaScript</li>
+      <li class="skill skill-about">Ruby</li>
+      <li class="skill skill-about">HTML</li>
+      <li class="skill skill-about">CSS</li>
+    </ul>
+    
+  </div>
+</div> 
+
+<div class="contact-1">
+  <div class="contact-full">
+    <div class="background-img1"></div>
+    <div class="background-img2"></div>
+    <div class="contact-para">
+      <h1 class="sp9">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
+    </div>
+      <div class="contact-mail">
+        <form action="https://formspree.io/f/xjvqvjew" method="POST">
+          <div class="input-none">
+            <input class="input-form input-name" maxlength="30" type="text" name="name" placeholder="Full name" required>
+          </div>
+         
+          <input class="input-form input-name" maxlength="30" type="text" name="name" placeholder="Full name" required>
+          <input class="input-form email-input" type="email" name="email" placeholder="email adress"  required>
+          <textarea class="text-name" maxlength="500" type="text" name="message"  cols="30" rows="10" placeholder="Hey i'd like to connect to.." required></textarea>
+    
+        <div class="d-btn">
+          <button class=" btn2" type="submit">Get in touch</button>
+        </div>
+        </form>
+        <div class="fin-p">
+          <p><a href="mailto:aminiroland01@gmail.com">aminiroland01@gmail.com</a></p>
+        </div>
```

Comment:
For your `CONTACT ME` section, kindly implement these changes :

- The text on the left-hand side should be in three lines as it is in the design given and kindly use the right alignment for it 😄 
- kindly add all thee background images present in the design given for this section of your project
- please add the horizontal line close to the footer icons and move the footer icons close to the end of the page

Your design :
![image](https://github.com/RolandAmini/portfolio-mobile/assets/20537712/645650ba-288f-4d49-8cae-3811e6b7d380)

Figma design :
![image](https://github.com/RolandAmini/portfolio-mobile/assets/20537712/183522ad-bca1-43d5-993b-98158f6cbd2d)